webapp/packages/plugin-administration/src/Administration/Administration.tsx

```diff
@@ -139,21 +138,21 @@ export const Administration = observer<React.PropsWithChildren<Props>>(function
           </TabList>
           <content-container ref={contentRef} as="div">
             {children}
-            <SlideBox open={optionsPanelService.active}>
-              <SlideElement>
+            <SlideBox>
+              <SlideElement open={optionsPanelService.active}>
                 <Loader suspense>
                   <content>
                     <OptionsPanel />
                   </content>
                 </Loader>
               </SlideElement>
-              <SlideElement>
+              <SlideElement open={optionsPanelService.active}>
                 <Loader suspense>
                   <content>
                     <ItemContent activeScreen={activeScreen} configurationWizard={configurationWizard} />
                   </content>
                 </Loader>
-                <SlideOverlay onClick={() => optionsPanelService.close()} />
+                <SlideOverlay open={optionsPanelService.active} onClick={() => optionsPanelService.close()} />
```

Comment:
Something is wrong; please share `.open` class between SlideBox SlideElement and SlideOverlay